### PR TITLE
bfcfg: support "Enable I2C0" system configuration

### PR DESCRIPTION
**Pull request by kblaiech**

### CODE REVIEW

File: bfcfg [hunk 1]
```diff
@@ -131,7 +131,7 @@ sys_cfg_one_byte()
 #   SYS_ENABLE_SPCR      31            1
 #   SYS_DISABLE_PCIE     32            1
 #   SYS_ENABLE_OPTEE     33            1
```
Comment (lsun100) [inline bfcfg:133]: Could you fix these three as well (adding 1)?

File: bfcfg [hunk 2]
```diff
@@ -154,7 +154,7 @@ sys_cfg()
   sys_cfg_one_byte ${tmp_file} "ENABLE_SPCR" 31 "${SYS_ENABLE_SPCR}"
   sys_cfg_one_byte ${tmp_file} "DISABLE_PCIE" 32 "${SYS_DISABLE_PCIE}"
   sys_cfg_one_byte ${tmp_file} "ENABLE_OPTEE" 33 "${SYS_ENABLE_OPTEE}"
```
Comment (lsun100) [inline bfcfg:156]: same here

**Review by lsun100 — CHANGES_REQUESTED** [review]

File: bfcfg (hunk 1 above)
Comment (kblaiech) [reply bfcfg:133]: done

File: bfcfg (hunk 2 above)
Comment (kblaiech) [reply bfcfg:156]: done

**Review by lsun100 — APPROVED** [review]

**Pull request merged**